packages/tailwindcss/src/utilities.ts

```diff
@@ -2785,6 +2792,699 @@ export function createUtilities(theme: Theme) {
     position: (value) => [gradientStopProperties(), decl('--tw-gradient-to-position', value)],
   })
 
+  if (enableMaskUtilities) {
+    /**
+     * @css `mask-image`
+     */
+
+    staticUtility('mask-none', [['mask-image', 'none']])
+
+    utilities.functional('mask', (candidate) => {
+      if (!candidate.value) return
+      if (candidate.modifier) return
+      if (candidate.value.kind !== 'arbitrary') return
+
+      // Arbitrary values
+      let value: string | null = candidate.value.value
+      let type =
+        candidate.value.dataType ??
+        inferDataType(value, ['image', 'percentage', 'position', 'bg-size', 'length', 'url'])
+
+      switch (type) {
+        case 'percentage':
+        case 'position': {
+          if (candidate.modifier) return
+          return [decl('mask-position', value)]
+        }
+        case 'bg-size':
+        case 'length':
+        case 'size': {
+          return [decl('mask-size', value)]
+        }
+        case 'image':
+        case 'url':
+        default: {
+          return [decl('mask-image', value)]
+        }
+      }
+    })
+
+    /**
+     * @css `mask-composite`
+     */
+
+    staticUtility('mask-add', [['mask-composite', 'add']])
+    staticUtility('mask-subtract', [['mask-composite', 'subtract']])
+    staticUtility('mask-intersect', [['mask-composite', 'intersect']])
+    staticUtility('mask-exclude', [['mask-composite', 'exclude']])
```

Comment:
These generate duplicate properties. Might be a Lightning CSS bug:

<img width="434" alt="image" src="https://github.com/user-attachments/assets/20166991-5785-4cb1-8a1b-35ee4f7c210f" />

Reply:
Yeah, this is a lightning thing. Need to file a bug for it.

Reply:
I'll make a note to do this today or tomorrow